test/demo.html

```diff
@@ -25,9 +25,9 @@
 
     animation: {
       duration: 1750,
-      delay: 100
+      delay: 100,
       //delay: function(index){}
-
+      disable: true
```

Comment:
Thanks for the changes! Now that we're not using this parameter, can you remove it please?